README.md

```diff
@@ -55,13 +58,19 @@
 
     ![Screenshot_20200916_231045](https://user-images.githubusercontent.com/17693494/93422970-26d9f580-f872-11ea-870d-4406db20e9d5.png)
 
-- **How do I keep my Git fork up to date?** Add a new remote called `upstream` targeting the original repository. Check out more information on [this issue](https://github.com/EddieHubCommunity/support/issues/94).
-- **What is DevRel?** Check out the discussion on [this issue](https://github.com/EddieHubCommunity/support/issues/64) or on [this YouTube video](https://www.youtube.com/watch?v=iUZerHctTB8&t=1534s).
-- **How do I get started with open source?** You can make your first pull request in this [repository](https://github.com/EddieHubCommunity/hacktoberfest-practice). Follow the instructions in the README, and don't forget – contributing to open source is not just about code – it's about collaboration, communication, and adding value. Don't be afraid to open an issue with a question :smiley:. If you have any trouble, you can ask questions in our Discord. We have a channel for `first-timers` and `opensource`.
+- **Q4 How do I keep my Git fork up to date?** 
+- **Ans:** Add a new remote called `upstream` targeting the original repository. Check out more information on [this issue](https://github.com/EddieHubCommunity/support/issues/94).
+  
+- **Q5 What is DevRel?** 
+- **Ans:** Check out the discussion on [this issue](https://github.com/EddieHubCommunity/support/issues/64) or on [this YouTube video](https://www.youtube.com/watch?v=iUZerHctTB8&t=1534s).
+  
+- **Q6 How do I get started with open source?**
+- **Ans:** You can make your first pull request in this [repository](https://github.com/EddieHubCommunity/hacktoberfest-practice). Follow the instructions in the README, and don't forget – contributing to open source is not just about code – it's about collaboration, communication, and adding value. Don't be afraid to open an issue with a question :smiley:. If you have any trouble, you can ask questions in our Discord. We have a channel for `first-timers` and `opensource`.
 
-  Here is a list of resources that also might help you start contributing to open source:
+  **Here is a list of resources that also might help you start contributing to open source:**
```

Comment:
Hi @PranavJoshi18! :) Congratulations on doing your first PR! 🎉🎊 So far, the info you placed here is great. I think what you can also do is add links to other videos on Open Source that are outside of Eddie’s channel. Here’s an example:

[Creating Forks and Pull Requests](https://youtu.be/nT8KGYVurIU)
My other recommendation is to add brief sentences about key moments to look for.

This is my first PR review, so I hope you find my feedback helpful. If you have any questions or feedback on my review, don’t hesitate to tag my name here!

Reply:
@CBID2 Hey! actually I made changes to the FAQ Section like adding question number tag, etc which I think you didnt notice!
Kindly check and merge the PR :)

Reply:
Hi @PranavJoshi18! I can review it, but I'm not granted merging rights.